lib/src/widgets/utils/ds_chip.widget.dart

```diff
@@ -11,7 +11,7 @@ class DSChip extends StatelessWidget {
   final VoidCallback? onTap;
   final Widget? trailingIcon;
   final EdgeInsetsGeometry? textPadding;
-  final double? width;
+  final double? size;
```

Comment:
What if we use a `Size` class instead of `double`, if someday we have the need of use `height` and `width` with different values?